ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -518,8 +577,61 @@ const NetworksForm = ({
     [t],
   );
 
+  const validateNetworkName = useCallback(
+    async (formChainId, formName) => {
+      let warningKey;
+      let warningMessage;
+      const decimalChainId = getDisplayChainId(formChainId);
+
+      if (!decimalChainId || !formName) {
+        setSuggestedNames([]);
+        return null;
+      }
+
+      if (safeChainsList.current.length === 0) {
+        warningKey = 'failedToFetchTickerSymbolData';
+        warningMessage = t('failedToFetchTickerSymbolData');
+      } else {
+        const matchedChain = getMatchedChain(
+          decimalChainId,
+          safeChainsList.current,
+        );
+
+        const matchedNames = getMatchedNames(
```

Comment:
When I try to add chain id 1 for example, it suggested a network name of "Ether".  Which is the token name but not the network name.  I'm wondering if it should lookup `.name` instead of `.nativeCurrency.name` in the chain id json.

<img width="392" alt="Screenshot 2024-06-13 at 10 57 48 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/8c94187c-5426-4285-b4eb-1233aaba984e">